src/zenml/pipelines/base_pipeline.py

```diff
@@ -588,3 +594,22 @@ def _read_config_steps(
                             parameter, step_name, previous_value, value
                         )
                     )
+
+    def get_runs(self):
```

Comment:
Nice idea to put this here. Can we potentially make this a `classmethod` so that users don't even have to create a pipeline instance for this?

Reply:
definitely